api/v1beta1/grafanaalertrulegroup_types.go

```diff
@@ -54,6 +54,9 @@ type GrafanaAlertRuleGroupSpec struct {
 
 	// +optional
 	AllowCrossNamespaceImport *bool `json:"allowCrossNamespaceImport,omitempty"`
+
+	// +optional
```

Comment:
```suggestion
        // Whether to enable or disable editing of alerts in the group in the Grafana UI
	// +optional
```

Reply:
Agree that `editable` is more readable. I used `disableProvenance` just to keep consistent with other provisioning methods like [terraform](https://grafana.com/docs/grafana/latest/alerting/set-up/provision-alerting-resources/terraform-provisioning/#enable-editing-resources-in-the-grafana-ui).

Reply:
I'm not against the Provenance name, I'm mainly suggesting adding a comment which will be added to the CRDs generated from the struct, that way I have something to search for

Reply:
Got it, will add the comment